include/odp/api/spec/event_types.h

```diff
@@ -52,6 +55,8 @@ extern "C" {
  *     - Timeout event (odp_timeout_t) from a timer
  * - ODP_EVENT_IPSEC_STATUS
  *     - IPSEC status update event (odp_ipsec_status_t)
+ * - ODP_EVENT_VECTOR
```

Comment:
Would the scheduled event always be ODP_EVENT_VECTOR or if odp_event_vector_config_t.event_type == ODP_EVENT_PACKET is it expeceted to be  ODP_EVENT_PACKET_VECTOR

Reply:
In this case the event type would always be `ODP_EVENT_VECTOR`. I'd like to keep the event and packet vector APIs separate. Optimally, the whole old packet vector API would be deprecated, but we cannot do that without breaking backwards compatibility.

Reply:
ack